resources/interaction-menu/cl_interaction.lua

```diff
@@ -1306,6 +1306,14 @@ Citizen.CreateThread(function()
 	end
 end)
 
+-- disable auto lock when using controllers
+Citizen.CreateThread(function()
+	while true do
+	  Citizen.Wait(0)
```

Comment:
```suggestion
	  Citizen.Wait(100)
```
for performance reasons, u can check `F8 > resmon` to see how necessary it is (if the number goes up when set to wait 0); but for now, to be safe, let's bump this up to every 100 frames instead of every frame just in case

And u can/should test to confirm it still works with this change, even tho I believe it will